connect_bi_reporter/credentials/models.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+import sqlalchemy as db
+
+from connect_bi_reporter.db import Model
+
+
+class Credential(Model):
+    __tablename__ = "credential"
+
+    PREFIX = 'CRED'
+
+    id = db.Column(db.String(20), primary_key=True)
+    name = db.Column(db.String(64))
+    connection_string = db.Column(db.String(256))
```

Comment:
For now I am fine, just curious:
shall connection be the string always? Or maybe some structured field is needed instead or in addition?

Reply:
Is true that connection can be constructed base on account_name and account_key, but from azure interface user can copy and paste it directly to the extension as "connection string", so account and key will be already included:
![image](https://github.com/cloudblue/connect-bi-reporter/assets/83503872/a66b66b3-192e-43bc-93f3-253466d0cf9a)